unit_tests/kernels/UnitTestKernelUtils.h

```diff
@@ -557,6 +557,84 @@ class SSTKernelHex8Mesh : public LowMachKernelHex8Mesh
   ScalarFieldType* fOneBlend_{nullptr};
 };
 
+/** Test Fixture for the Turbulence Kernels
+ *
+ */
+class TurbulenceKernelHex8Mesh : public LowMachKernelHex8Mesh
```

Comment:
The Rodi term is going to be used with `k-sgs` model, so can we just extend the `KsgsKernelHex8Mesh` instead of creating an entirely new class here?

Reply:
Uh, I'll have to look at what the KsgsKernelHex8Mesh is.......